src/main/java/com/amazon/asksdk/cloudfoody/Restaurant.java

```diff
@@ -0,0 +1,102 @@
+package com.amazon.asksdk.cloudfoody;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Restaurant extends OpenTable {
+
+    private String name, address, city, state, area, country, phone, reserveURL, mobileReserveURL, imageURL, postalCode;
```

Comment:
NP: please split into several lines to improve readability.

Reply:
is it better to put every single variable on a separate line or just split them into 3 lines?

Reply:
3 lines would be OK.

Reply:
Done!